javascript/test/control-expected/sample-featureFlag.js

```diff
@@ -0,0 +1,25 @@
+// Simple flag cleanup in conditional
+f2();
+
+// ---------------------------------
+
+// Assignment cleanup
+f1();
+
+// ----------------------------------
+
+// function cleanup
+
+if(f1()) {
+   f1()   
+} else {
+   f2()   
+}
+
+// ----------------------------------
+
+// Complex cleanup
+var c = f1();
+
+// Literal conversion
+console.log(false);
```

Comment:
We plan to eventually run a code formatter on all files. You can ignore minor code styling issues in your review.

Reply:
Please do run the auto-formatter before landing this PR (probably would have been good to run it before reviews too, for readability, but not sure if it will mess existing PR comments at this point). Also, consider integrating the auto-formatter to run on every commit and check in CI as a follow up PR (not on this one, though, this is already a large code change 🙂 )

Reply:
May be https://github.com/prettier/prettier?

Reply:
Done 78adb312677f136e9f7d41027305280007494fd8.